<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LzGraspClass</class>
 <widget class="QWidget" name="LzGraspClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>兰州双目标定采集程序</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>亮度阈值</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="graythresh">
           <property name="text">
            <string>30</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>圆斑尺寸</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="blobsize_min">
           <property name="text">
            <string>20</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="blobsize_max">
           <property name="text">
            <string>200</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>左相机序列号</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="camseq_left"/>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>右相机序列号</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="camseq_right"/>
         </item>
         <item>
          <widget class="QComboBox" name="camgroup_sellect"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>标定板尺寸</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="patternsize_col">
           <property name="text">
            <string>5</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="patternsize_row">
           <property name="text">
            <string>5</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>曝光时间(us)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="exposuretime">
           <property name="text">
            <string>5000</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="confirmbutton">
       <property name="text">
        <string>确定设置</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="img_left">
       <property name="minimumSize">
        <size>
         <width>351</width>
         <height>401</height>
        </size>
       </property>
       <property name="text">
        <string>左图</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="img_right">
       <property name="minimumSize">
        <size>
         <width>351</width>
         <height>401</height>
        </size>
       </property>
       <property name="text">
        <string>右图</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="graspbutton">
         <property name="text">
          <string>采集(Space)</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="savebutton">
         <property name="text">
          <string>存储(Enter)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
